hummingbird/ml/operator_converters/_tree_implementations.py

```diff
@@ -48,19 +47,6 @@ def aggregation(self, x):
         """
         pass
 
-    @abstractmethod
-    def calibration(self, x):
```

Comment:
can you explain why this isn't needed (so that i understand)?  Is this related to #218 ?

Reply:
I pushed everything into the aggregation function. In this way we don't have to use two different functions to change the behavior of trees. In calibration I was basically putting the onnx `post_transform`. DecisionTree implementations don't need post transform, while the post_transforms for GBDT implementations now are inside aggregation.